features/step_definitions/file_steps.coffee

```diff
@@ -5,17 +5,21 @@ path = require 'path'
 module.exports = ->
 
   @Given /^I have no test files$/, (done) ->
-    done() # No op for readability
+    done()
 
 
-  @Given /^I have 1 (passing|failing) test$/, (status, done) ->
-    outputTestFile path.join(@tmpDir, 'example_spec.coffee'), passing: status is 'passing', done
+  @Given /^I have a (passing|failing) test$/, (status, done) ->
+    outputTestFile path.join(@tmpDir, 'example_spec.coffee'),
+                   passing: status is 'passing',
+                   done
```

Comment:
Regarding mixing keywords args with normal ones: That's indeed a mixed bag in my version. From a pragmatic perspective I'd say (2) is preferred, because its the most compact and readable (imo). If we run into line length restrictions, (3) is the next best solution that should be tried here. Because it visually separates the different arguments better than (1). If that is still to wide, then fall way back to (1). 

(1) is really poorly readable imo, because this really looks like 3 parameters, where it actually is only two, with the first one spanning two lines. There is no syntax that indicates this, it has to be derived by the reader from the data type of the parameters. Boo on that!

My proposal is admittedly not as simple as "always use (1)", as you propose. But it minimizes the dangerous overlapping use of too similar-looking code constructs in the code base. Which is valuable. And excessive parameteritis that requires such long parameter lists is an indicator that there is maybe a parameter object/class waiting to be extracted, or a refactoring that wants to happen here. 

What I describe is not arbitrary. At Google there are many such alternative formatting guidelines. The reason is that producing beautiful, intuitive, and well readable code is more a product of art, craft, and expertise than blindly following simple rules, and using only a single tool. Similar to code architecture, its best to try to keep things simple, elegant, and concise, and work our way around the few edge cases that exist.